### ISSUE TITLE
[Designer] [Bug 40018278](https://microsoft.visualstudio.com/OS/_workitems/edit/40018278): [ActivityUpdate.json]: Screen reader is not announcing descriptive information for <"Next"/ "Previous"> icons button.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7615]
### Problem Description

[Bug 40018278](https://microsoft.visualstudio.com/OS/_workitems/edit/40018278): [ActivityUpdate.json]: Screen reader is not announcing descriptive information for <"Next"/ "Previous"> icons button.

Test Environment:
OS: Win11 (25136.1000)
UWP App version: 3.0.2206.2001
Screen reader: Narrator

Repro Steps:
1. Run AC UWP Visualizer App.
2. Turn on the narrator.
3. Open 'activityupdate.json' code template and visit the card.
4. Get focus on calendar UI.
5. Get focus on Next and previous buttons.
6. Observe the issue.

Actual Result:
Screen reader is not announcing descriptive information for <"Next"/ "Previous"> icons button.

Expected Result:
Screen reader should announce information like as "Next month"/"Previous month" button.

User Experience:
Screen reader users will find it difficult to interact with the UI if no description is provided.

**"Have feedback to share on Bugs ? Please help fill [[Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0)]

### Card JSON

```JSON
Designer Bug. Refer MS App Center builds.
```

**Comment by vagpt** [comment]
This issue is still repro at our end.
